src/Integration.Vsix/SonarLintTagger/TextBufferIssueTracker.cs

```diff
@@ -275,7 +274,9 @@ void IIssueConsumer.Accept(string path, IEnumerable<Issue> issues)
                 return;
             }
 
-            issues = issuesFilter.Filter(path, issues);
+            // TODO: collect the additional information required to compare be able to filter
+            // the issues (i.e. the text of the line and the line hash)
+            // issues = issuesFilter.Filter(path, issues);
```

Comment:
Next steps: the `TextBufferIssueTracker` class has access to the underlying text buffer so it can find the text for the line.